spike/main.py

```diff
@@ -1,6 +1,10 @@
 import tkinter as tk
 from ui import Application
 
+def print_file():
+    f = open("inputfile.txt", "r")
+    print(f.read())
+    f.close()
```

Comment:
This definitely works. However, there is a more Pythonic way of doing it utilizing the `with` keyword. It looks like:
```python
with open(path, 'r') as f:
   # do stuff #
```
It would also be useful to not hardcode the filepath and to take it as a parameter for `print_file`.